CHANGELOG.md

```diff
@@ -11,25 +11,36 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 ### Added
 
 - Add `WithMetricsBuiltinsEnabled()` option and environment variable
-  `LS_METRICS_BUILTINS_ENABLED`, which defaults to true.  When metrics
-  builtins are enabled,
-  [runtime](https://github.com/open-telemetry/opentelemetry-go-contrib/tree/main/instrumentation/runtime)
-  and
-  [host](https://github.com/open-telemetry/opentelemetry-go-contrib/tree/main/instrumentation/host)
-  metrics instrumentation will be reported automatically.
+  `LS_METRICS_BUILTINS_ENABLED`, which defaults to true.
   [#265](https://github.com/lightstep/otel-launcher-go/pull/265)
-- Proposed replacement for go-contrib instrumentation/runtime added as lightstep/instrumentation/runtime.
+- New replacement for go-contrib instrumentation/runtime added as lightstep/instrumentation/runtime.
```

Comment:
one minor thing: can you add a note on what existing users of the previous `instrumentation/runtime` need to do or be aware of (if anything) when making the switch.

Reply:
Done. (FWIW, this was in the README already, now added in the CHANGELOG).